src/browser/documentation/index.ts

```diff
@@ -188,25 +338,41 @@ export default {
   },
   play: {
     title: 'Guides & Examples',
+    chapters: {
+      concepts: playConcepts,
+      cypher: playCypher,
+      iconography: playIconography,
+      intro: playIntro,
+      learn: playLearn,
+      movieGraph: playMovieGraph,
+      'movie-graph': playMovieGraph,
+      movies: playMovieGraph,
+      northwind: playNorthwindGraph,
+      'northwind-graph': playNorthwindGraph,
+      northwindGraph: playNorthwindGraph,
+      start: playStart,
+      typography: playTypography,
+      unfound: playUnfound,
+      writeCode: playWritecode
+    }
+  },
+  // Guides are play-guides but migrated to be viewable in the sidebar
+  guide: {
+    title: 'Built-in Browser guides',
     chapters: {
       concepts: guideConcepts,
       cypher: guideCypher,
+      index: guideIndex,
       intro: guideIntro,
-      learn: guideLearn,
-      movieGraph: guideMovieGraph,
       movies: guideMovieGraph,
-      northwindGraph: guideNorthwindGraph,
+      movieGraph: guideMovieGraph,
+      'movie-graph': guideMovieGraph,
       northwind: guideNorthwindGraph,
-      iconography: guideIconography,
-      start: guideStart,
-      typography: guideTypography,
-      unfound: guideUnfound,
-      writeCode: guideWritecode,
-      // Commands only
-      'query-template': {
```

Comment:
was this removed on purpose? I don't see it in the new `play` section you created.